app/Http/Controllers/Product/ProductController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Http\Controllers\Product;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Product;
+use App\Brand;
+use App\Category;
+
+class ProductController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $products = Product::all()->where('seller_id',auth()->id());
+        return view('seller.product.index_product')->with('products',$products);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $brands = Brand::get_available_brands();
```

Comment:
What is so special about this method get_available_brands() ???

you could have just used Brand::all(); or Brand::get();